cmd/litcli/autopilot.go

```diff
@@ -216,11 +223,27 @@ func initAutopilotSession(ctx *cli.Context) error {
 		}
 	}
 
+	configs := ctx.StringSlice("configuration")
+	features := ctx.StringSlice("feature")
+
+	if len(configs) > 0 && len(features) != len(configs) {
+		return fmt.Errorf("number of features and configurations " +
+			"must match")
+	}
+
 	featureMap := make(map[string]*litrpc.FeatureConfig)
-	for _, feature := range ctx.StringSlice("feature") {
+	for i, feature := range ctx.StringSlice("feature") {
+		var config []byte
+
+		// We allow empty configs, to signal the usage of the default
+		// configuration when the session is registered.
```

Comment:
added all the suggestions, hope we have a better description now :). is it a breaking change? we can leave out all config flags and it will use the defaults

Reply:
ah - you are right. Sorry about that 🙈